### PR TITLE
Architecture diagram fixes

### PR DESCRIPTION
**Pull request by tsenart**

### CODE REVIEW

**Review by efritz — APPROVED** [review]

**Comment by efritz** [comment]
Can you regenerate the svg?

File: doc/dev/background-information/architecture/architecture.dot [hunk 1]
```diff
@@ -238,6 +239,7 @@ digraph architecture {
     searcher -> gitserver [fillcolor="#e2a8fd"]
     symbols -> gitserver [fillcolor="#c1eaea"]
     zoekt_indexserver -> {frontend, gitserver} [fillcolor="#aaccff"]
+    query_runner -> frontend [fillcolor="#aaffff"]
```
Comment (efritz) [inline doc/dev/background-information/architecture/architecture.dot:242]: Is this for more than config?

Comment (tsenart) [reply doc/dev/background-information/architecture/architecture.dot:242]: Yeah, it's listing all saved queries.

**Comment by tsenart** [comment]
> Can you regenerate the svg?

The query runner link back to the frontend doesn't work. I'm not sure why.

**Comment by tsenart** [comment]
@efritz: https://github.com/sourcegraph/sourcegraph/blob/53c418d32bf4cf3852e2ea24b6c24342bf74374d/doc/dev/background-information/architecture/architecture.svg

Any clue why?

**Pull request closed without merging**